.github/workflows/ci-schema.yml

```diff
@@ -0,0 +1,13 @@
+name: Check schema
+
+on:
+  push:
+    branches: [master, "release/*"]
+  pull_request:
+    branches: [master, "release/*"]
+
+jobs:
+  check-schema:
+    uses: Lightning-AI/utilities/.github/workflows/check-schema.yml@v0.4.0
+    with:
+      azure-dir: ''
```

Comment:
Given that Microsoft doesn't provide the correct schema file as I commented in https://github.com/Lightning-AI/lightning-flash/pull/1455#issuecomment-1244793607, I'd like to suggest skipping all azure configs for the time being.